.github/workflows/java-cd.yml

```
@@ -102,7 +102,7 @@ jobs:
             - name: Install protoc (protobuf)
               uses: arduino/setup-protoc@v3
               with:
-                  version: "26.1"
+                  version: "28.1"
```

28.2, 27.5, and 25.5 all came out to address the vulnerability. 
```suggestion
                  version: "28.2"
```